x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/original_scenarios.md

```diff
@@ -0,0 +1,264 @@
+# Installation and Upgrade of Prebuilt Rules
+
+## Scenarios
+
+### Notifications
+
+#### **Scenario: No callout messages are displayed when there are no pending installs/updates**
+
+```Gherkin
+Given user has the latest version of prebuilt rules <prebuilt_rules_status>
+When user navigates to the Rule Management page
+Then no callout message is displayed for <prebuilt_rules_status> rules
+And no badges with number of available rules to install/update are displayed
+
+Examples:
+  | prebuilt_rules_status |
+  |            to install |
+  |             to update |
+```
+
+#### **Scenario: Callout message is displayed when there are new prebuilt rules available to install**
+
+```Gherkin
+Given user already has 8.x prebuilt rules installed
+And there are new prebuilt rules available to install
+When user navigates to the Rules Management Page
+Then user should see a callout message to install new prebuilt rules
+And the number of new rules available to install should be displayed on the +Add Elastic Rules link
+```
```

Comment:
Replaced by:

#### **Scenario: User is notified when some prebuilt rules can be installed**

**Coverage**: 1 e2e test with mock rules + 1 integration test.

```Gherkin
Given X prebuilt rules are installed in Kibana
And there are Y more prebuilt rules available to install
And for all X installed rules there are no new versions available
When user opens the Rule Management page
Then user should see a CTA to install prebuilt rules
And user should see the number of rules available to install (Y)
And user should NOT see a CTA to upgrade prebuilt rules
And user should NOT see a number of rules available to upgrade
And user should NOT see the Rule Updates table
```